packages/twenty-front/src/modules/ui/utilities/scroll/components/ScrollWrapper.tsx

```diff
@@ -15,9 +15,12 @@ import { scrollWrapperScrollTopComponentState } from '@/ui/utilities/scroll/stat
 import { useSetRecoilComponentStateV2 } from '@/ui/utilities/state/component-state/hooks/useSetRecoilComponentStateV2';
 import 'overlayscrollbars/overlayscrollbars.css';
 
-const StyledScrollWrapper = styled.div<{ scrollHide?: boolean }>`
+const StyledScrollWrapper = styled.div<{
+  scrollHide?: boolean;
+  fullHeight?: boolean;
+}>`
   display: flex;
-  height: 100%;
+  height: ${({ fullHeight }) => (fullHeight ? '100%' : 'fit-content')};
```

Comment:
I think we should see with @ehconitin here

Reply:
See my comment above https://github.com/twentyhq/twenty/pull/9162#discussion_r1900128828